src/App/Fossa/ArchiveUploader.hs

```diff
@@ -3,6 +3,7 @@
 module App.Fossa.ArchiveUploader (
   archiveUploadSourceUnit,
   arcToLocator,
+  compressFile,
```

Comment:
At some point soon we ought to move utilities like this to a common location so that the scanner doesn't depend on the uploader.  I'll let you make the call on whether that is now or not.